auto-gen-mr-status.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+import os
+import sys
+import re
+
+output_filename='documentation/sphinx/source/mr-status-auto-gen.rst.inc'
+
+print "WARNING: This is a very simple script to help keep machine-readable JSON status document synchronize with Schemas.cpp"
+print "WARNING: THIS SCRIPT DOES NOT HANDLE COMMENTS FOR KEYS WITH THE SAME NAME!"
+
+# Read old machine-readable status document mr-status-old.rst to get the comment
+# Store the comment for each keyword into a dictionary
+print "Read comment from old mr status file documentation/sphinx/source/mr-status-json-old.rst";
```

Comment:
I don't think we'll want to keep around the old documentation file just for parsing comments out of it, because at the very least that's going to make adding new comments awkward. If we need to define comments externally from the schema, my suggestion would be to create a new document that acts as a map between fully qualified paths and the associated comments.

Since the schema writer isn't path aware at the moment, you could then discard all but the last element of the path when you import it into this script and warn if there are duplicates.

Reply:
Just to confirm my understanding is correct:
I will generate a separate file that contains the map between the keyword (I,e., last element of the path) and the comment. 
I will remove the old mr-status-old.rst
Is that correct?

Reply:
No, the separate file would contain a mapping from full path to comment. The script would throw away most of the path for now.

Reply:
Also for what it's worth, I'm not sure I love this solution of having a comment map in a separate file, but it seems better than having our entire old documentation for that purpose. If you have a better idea that's appropriate for this quick solution, then feel free to do that instead.